parent/pom.xml

```diff
@@ -277,6 +277,45 @@
 
   <dependencyManagement>
     <dependencies>
+
+      <dependency>
+        <groupId>com.squareup.okhttp3</groupId>
+        <artifactId>okhttp</artifactId>
+        <version>4.11.0</version>
+      </dependency>
+
+      <dependency>
+        <groupId>io.opentelemetry.instrumentation</groupId>
+        <artifactId>opentelemetry-zipkin-spring-boot-starter</artifactId>
+        <version>2.20.1-alpha</version>
+      </dependency>
+
+      <dependency>
+        <groupId>io.opentelemetry.instrumentation</groupId>
+        <artifactId>opentelemetry-grpc-1.6</artifactId>
+        <version>2.20.1-alpha</version>
+      </dependency>
+
+      <dependency>
+        <groupId>com.google.cloud.opentelemetry</groupId>
+        <artifactId>detector-resources-support</artifactId>
+        <version>0.36.0</version>
+      </dependency>
+
+      <dependency>
+        <groupId>io.opentelemetry.contrib</groupId>
+        <artifactId>opentelemetry-gcp-resources</artifactId>
+        <version>1.49.0-alpha</version>
+      </dependency>
+
+      <dependency>
+        <groupId>io.opentelemetry.instrumentation</groupId>
+        <artifactId>opentelemetry-instrumentation-bom</artifactId>
+        <version>2.20.1</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
```

Comment:
🔧 Let's figure out which dependencies we actually need to keep. Fex, zipkin should be obsolete by now.

We also included some alpha dependencies which is generally not a great idea.

Reply:
Cleaned up in https://github.com/camunda/camunda/pull/38884/commits/de48145d595380c2379f3e53135e1898d24f741d

The grpc alpha dependency is in use. We should find an alternative so we don't use an alpha version.